### PR TITLE
T3 collections

### PR DESCRIPTION
**Pull request by emmaalecrim**
Relevant Links:
https://docs.oracle.com/javase/tutorial/collections/interfaces/index.html
https://docs.oracle.com/javase/tutorial/collections/interfaces/list.html
https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html#constructor.summary

### CODE REVIEW

File: src/s1/t3/n1/exercici1/Main.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package s1.t3.n1.exercici1;
+
+
+
+public class Main {
+    void main(String[] args) {
+        Month august = new Month(Month.MONTHS.august);
+        // Init year empty and fill manually according to instruction.
```
Comment (marinaaguiar) [inline src/s1/t3/n1/exercici1/Main.java:8]: You should remove the comments. 
Too many comments mean that your code is not so readable... that was what both teachers have said.

File: src/s1/t3/n1/exercici1/Month.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    };
+    public final String name;
+    public final int ordinal;
+
+    public Month(MONTHS month) {
+        this.name = month.toString();
+        this.ordinal = month.ordinal();
+    }
+
+    @Override
```
Comment (marinaaguiar) [inline src/s1/t3/n1/exercici1/Month.java:29]: I didn't understand the necessity for this... Could you explain me?

File: src/s1/t3/n1/exercici3/CountriesAndCapitalsQAGame.java [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        BufferedWriter writer = new BufferedWriter(new FileWriter("./src/s1/t3/n1/exercici3/classificacio.txt", true));
+        writer.write(username + " " + score);
+        writer.newLine();
+        writer.close();
+        System.out.println("Score saved.");
+    }   
+}
+
+
+    // void main(String[] args) {
```
Comment (marinaaguiar) [inline src/s1/t3/n1/exercici3/CountriesAndCapitalsQAGame.java]: You should clean unused code.

**Review by marinaaguiar — CHANGES_REQUESTED** [review]

File: src/s1/t3/n1/exercici1/Main.java (hunk 1 above)
Comment (emmaalecrim) [reply src/s1/t3/n1/exercici1/Main.java:8]: It is (documenting) that it's an antipattern and why.

File: src/s1/t3/n1/exercici1/Month.java (hunk 2 above)
Comment (emmaalecrim) [reply src/s1/t3/n1/exercici1/Month.java:29]: See HashSet<Month>. These overrides define equality logic.

File: src/s1/t3/n1/exercici3/CountriesAndCapitalsQAGame.java (hunk 3 above)
Comment (emmaalecrim) [reply src/s1/t3/n1/exercici3/CountriesAndCapitalsQAGame.java]: Great catch, thank you.

**Review by emmaalecrim — COMMENTED** [review]
Reviewed comments. I went overboard trying to understand the collections api a little bit deeper, which makes my code a bit more complex than the easiest/best implementation. 
Due to the nature of the exercises being demonstration/proof driven, I think it's not innapropriate.  Suggestions to making it more easily undestandable are open. Will ask for more eyes.

**Review by marinaaguiar — APPROVED** [review]

**Comment by emmaalecrim** [comment]
TODO: 
- [x] review access control

**Comment by emmaalecrim** [comment]
new feedback
[ ] separa los enums en archivos diferentes.

[ ] dile a un compañero que revise el código

**Pull request merged**